src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2589,11 +2627,11 @@ TEST_P(conv_fp32_reorder_bfyx_to_fsv32_conv_mean, have_mean) {
     execute(p);
 }
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_reorder_bfyx_to_fsv32_conv_mean, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  3, 3 }
+    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  4, 4 }
 }));
 
-
-class conv_fp32_reorder_bfyx_to_fsv32_conv_subtract : public ConvFusingTest {};
+// 'reorder_fsv32' should NOT be optimized because it has 'subtract'
```

Comment:
same as above. "should" sounds very strong.

Reply:
applied